docs/src/@primer/gatsby-theme-doctocat/nav.yml

```diff
@@ -46,8 +46,6 @@
   children:
     - title: Alerts
       url: /components/alerts
-    - title: Autocomplete
-      url: /components/autocomplete
```

Comment:
👍  On just removing the nav item for now.

[Internal discussion](https://github.com/github/primer/discussions/1019) around not having to maintain documentation in PCSS **AND** PVC.